django_cookies_samesite/middleware.py

```diff
@@ -33,7 +34,11 @@ def process_response(self, request, response):
         # same-site = None introduced for Chrome 80 breaks for Chrome 51-66 
         # Refer (https://www.chromium.org/updates/same-site/incompatible-clients)
         http_user_agent = request.META.get('HTTP_USER_AGENT') or " "
-        if re.search(CHROME_VALIDATE_REGEX, http_user_agent):
+        ua_string = http_user_agent
+	user_agent = parse(ua_string)
+	if user_agent.browser.family == "Safari":
+	    return response
+	if re.search(CHROME_VALIDATE_REGEX, http_user_agent):
```

Comment:
Is the alignment fine ?

Reply:
![Screenshot from 2020-03-24 14-18-02](https://user-images.githubusercontent.com/3314246/77406222-54217b80-6dda-11ea-868e-61ae3cefebca.png)

Reply:
looks fine to me.